### DISCUSSION TITLE
AI Ethics in MMI-BMI

### DISCUSSION THREAD

**Discussion by amritchhetrib78** [OpenMined/courses#71] [Feedback] [answered]
What are the Use Cases of AI Ethics in Machine Brain Interface related Solutions?

**Comment by yemikifouly** [comment]
Hello Amrit! 👋 @iamtrask Can you help here?

**Comment by znreza** [comment, marked as answer]
@amritchhetrib78 on top of the mind I can think of the following use-cases:

1. Fight racial bias: there is a reported bias to predict black people as convicts by softwares. However, this kind of bias is very much prevalent in human decision-making as well. Brain-computer interface or BCI can help detect if someone's decision is influenced by the other person's race. It can help the decision-maker too to keep themselves aware of any kind of racial bias.

2. In companies, your manager can check if you are distracted or focused at work by monitoring your brain activity while at work. However, there is a huge possibility of this data to be misused. This creates the bundling problem mentioned in the course. Using privacy preserving Structured Transparency, the bundling problem can be solved so that your manager only gets a notification of whether you are focused on the task assigned to you or not, not what you are focusing on at the moment.
